### PR TITLE
[fix](light-schema-change) fix schema consistency check failed

### PR DESCRIPTION
**Pull request by Hastyshell**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

File: regression-test/suites/partition_p0/test_partition_operation.groovy [hunk 1]
```diff
@@ -37,6 +37,7 @@ suite("test_partition_operation", "p1") {
             AGGREGATE KEY(k1,k2,k3,k4,k5,k6) 
             ${partitionInfo} 
             ${distributionInfo}
+            BUCKETS 8
```
Comment (yiguolei) [inline regression-test/suites/partition_p0/test_partition_operation.groovy:40]: why change all buckets to 8 ???

Comment (TangSiyang2001) [reply regression-test/suites/partition_p0/test_partition_operation.groovy:40]: To avoid problems like that not appear in test.

Comment (hello-stephen) [reply regression-test/suites/partition_p0/test_partition_operation.groovy:40]: line 72 already has 'BUCKETS 13'

File: be/src/service/internal_service.cpp [hunk 2]
```diff
@@ -693,8 +693,9 @@ void PInternalServiceImpl::_get_column_ids_by_tablet_ids(
             // check schema consistency, column ids should be the same
             const auto& columns = tablet->tablet_schema()->columns();
             std::vector<int32_t> column_ids(columns.size());
-            std::transform(columns.begin(), columns.end(), column_ids.begin(),
-                           [](const TabletColumn& c) { return c.unique_id(); });
+            for (const auto& col : columns) {
+                column_ids.push_back(col.unique_id());
+            }
```
Comment (dataroaring) [inline be/src/service/internal_service.cpp:698]: Column_ids should be type of set?

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 44.47 seconds
 stream load tsv:          553 seconds loaded 74807831229 Bytes, about 129 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          29.1 seconds inserted 10000000 Rows, about 343K ops/s
 storage size: 17162114407 Bytes

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]
LGTM

**Pull request merged**